pkg/cliplugins/crd/cmd/cmd.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2022 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+
+	"github.com/kcp-dev/kcp/pkg/cliplugins/crd/plugin"
+)
+
+var (
+	crdExample = `
+	# Convert a CRD in a yaml file to an APIResourceSchema. For a CRD named widgets.example.io, and a prefix value of
+	# 'today', the new APIResourceSchema's name will be today.widgets.example.io.
+	%[1]s crd snapshot -f crd.yaml --prefix 2022-05-07 > api-resource-schema.yaml
```

Comment:
does thi smean without `-f` we read a real CRD from the cluster?

Reply:
or can one do `kubectl get crd foo | kubectl crd snapshot -f -`?

Reply:
No, the current code requires -f

Reply:
Yes -f - is supported for stdin

Reply:
would add that example with the pipe here. It's useful.

Reply:
Fixed in force-push